src/main/java/com/adamk33n3r/runelite/watchdog/ui/notifications/panels/NotificationPanel.java

```diff
@@ -119,13 +124,15 @@ public NotificationPanel(Notification notification, NotificationsPanel parentPan
         afkTimerConfigRow.setBackground(ColorScheme.DARKER_GRAY_COLOR);
         JLabel afkTimerLabel = new JLabel("AFK Seconds:");
         afkTimerLabel.setToolTipText("Number of seconds for which the client doesn't get any mouse or keyboard inputs.");
+        AtomicInteger previousAFKSeconds = new AtomicInteger(notification.getFireWhenAFKForSeconds() > 0 ? notification.getFireWhenAFKForSeconds() : config.defaultAFKSeconds());
```

Comment:
Never heard of this, but I'm guessing it's some thread-safe thing? From what I understand, all swing components event stuff run on the Event Dispatch Thread (EDT) so there shouldn't be a need for this. Did you see any issue with just using an `int`? I would guess if it _could_ have problems, it would only be if you were changing the number while clicking the button at the same time? haha. 

Anyway, mostly just curious!

Reply:
When using a normal `int` and reassigning it, I got the error in IntelliJ:

> Variable used in lambda expression should be final or effectively final

Which then had the suggested action to "Convert to atomic" so that's what I did. I did a quick google and it seems like this is a precaution for variables in closures. But I'm not entirely sure.